warp/__init__.py

```diff
@@ -316,5 +316,6 @@
 from warp.math import *
 
 from . import config as config
+from . import types as types
```

Comment:
We consider anything in `__init__.py` to be part of the public API, and the `types` module is not something that we would like to expose as a whole.

Reply:
When I was reading documentation, I noticed that `wp.types` was used [here](https://nvidia.github.io/warp/modules/runtime.html#matrices), [here](https://nvidia.github.io/warp/codegen.html#customize-dimensions) and [here](https://github.com/NVIDIA/warp/blob/46dc8b6a17ae9ec381e8a133420e1bba8904ec00/warp/examples/fem/utils.py#L365), so I assumed that `wp.types` was part of the public API. If `wp.types` is not, I'll remove it from `__init__.pyi`.